index.html

```diff
@@ -584,7 +605,7 @@ <h1>Trust Model</h1>
 to be a correct record of which identifiers belong to which <a>entities</a>.
         </li>
         <li>
-The <a>subject</a> trusts the <a>issuer</a> to issue true (i.e. not false)
+The <a>subject</a> trusts the <a>issuer</a> to issue true
```

Comment:
We're in the section about how trust works. We don't trust a credential because the issuer "asserts certain types of information about the subject" -- we trust the credential because we think it's likely that the issuer's assertions are actually true. The verification of a credential does not, of course, test the truth value of assertions--only the authorship of them--but as a description of how trust works, merely stating that we trust a party to make assertions is not going far enough, I think.